src/main/java/com/shopify/model/MetafieldValueType.java

```diff
@@ -2,7 +2,10 @@
 
 public enum MetafieldValueType {
```

Comment:
Probably should make the breaking change here to MetafieldType instead of MetafieldValueType so it keeps in parallel with the shopify naming